### DISCUSSION TITLE
About the rate-distortion curve of SSF model

### DISCUSSION THREAD

**Discussion by OctoberKat** [InterDigitalInc/CompressAI#136] [General]
Hi, thanks for your great work!
Could you please release a picture of the rate-distortion curve based on the reimplemented scale-space flow model? I wonder if the model can achieve the performance reported in the paper. Thank you!

**Comment by OctoberKat** [comment]
I find that the reimplemented SSF performs slightly better at large bpp (> 0.3), but performs worse at small bpp.

**Reply by fracape** [reply]
Hi, you are right, I am running some more experiments. Please note that those high bitrates don't make much sense in terms of video coding (too high). That may be due in a large part to the training set that is not clearly documented. We performed training on vimeo90k with patches of 256x256 and the UGC dataset (384x384). Contrary to image compression, we could not reproduce the results directly with widely used datasets. I moved it to a discussion, rather than a bug issue. However, please file a bug if you find one in the code.

**Comment by herok97** [comment]
Hi OctoberKat, 
I got very lower result from compressai's SSF model than paper's.

So i'm confused with your discussion T_T 

Could you check my discussion? 😂
(https://github.com/InterDigitalInc/CompressAI/discussions/154)
